### ISSUE TITLE
Downloading

### ISSUE THREAD

**Issue by icss91** [marcelluseasley/MusicQuik#1]
Hey, the downloading function doesnt work,

however i suggest you use Jamendo as the music download source - 

https://developer.jamendo.com/v3.0 

they have open API that is legal to use and download the music on their platform.

App would look be really awesome if you'd implement it.

**Comment by icss91** [comment]
Or even better http://freemusicarchive.org/, due to jamendo's api is limited for a free user.

**Comment by marcelluseasley** [comment]
yeah I saw that. I will look into free music archive. thank you

On Thu, Jun 16, 2016, 7:11 PM icss91 notifications@github.com wrote:

> Or even better http://freemusicarchive.org/, due to jamendo's api is
> limited for a free user.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/marcelluseasley/MusicQuik/issues/1#issuecomment-226639916,
> or mute the thread
> https://github.com/notifications/unsubscribe/AACcuMEDk6AXdtMSsuuXfa4FVu0MrHtkks5qMdgygaJpZM4I33Pp
> .
> 
> ## 

## Marcellus Easley

https://www.linkedin.com/in/marcelluseasley

**Comment by marcelluseasley** [comment]
downloading does work

**Comment by icss91** [comment]
I'm quite sure amazon would let the app be, if you would make a version with http://freemusicarchive.org/ instead of the current database, it's quite sad they deleted the app, while you were getting more and more downloads daily :(

**Comment by marcelluseasley** [comment]
yeah I should do a version with freemusicarchive.org, but I don't know if the same appeal will be there.

**Comment by icss91** [comment]
You wont know if you dont try :), anyways best of luck!

**Comment by marcelluseasley** [comment]
Thank you!

**Comment by xcitydev** [comment]
> Or even better http://freemusicarchive.org/, due to jamendo's api is limited for a free user.
> 

Please any help or open source code on how this can be done?
